Bottom: 479e122cf112b707f6fd7e19da415358e34452d6
Top:    67a2863c181a32b7ed5eb0e86016a3eedef41bb2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-11 14:56:18 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
